libs/shared-entity/Cargo.toml

```diff
@@ -28,6 +28,8 @@ validator = { version = "0.16", features = ["validator_derive", "derive"], optio
 futures = "0.3.30"
 bytes = "1.6.0"
 log = "0.4.21"
+collab-folder = { workspace = true }
```

Comment:
can we remove collab-folder here?

Reply:
this is required and used (`Folder`, `ViewIcon`) for the returned folder structure (`FolderView`)

Reply:
this also required to represents the published data in the frontend, which the structs can then be reused

Reply:
move collab_folder_to_folder_view to get_user_workspace_structure then you don't need to depend on collab-folder

Reply:
removed `collab_folder` dep on `shared-entitiy`